python/tvm/relay/frontend/pytorch.py

```diff
@@ -4237,7 +4243,9 @@ def create_convert_map(self):
             "torchvision::deform_conv2d": self.deform_conv2d,
             "aten::unbind": self.unbind,
             "aten::__and__": self.logical_and,
+            "aten::__or__": self.logical_or,
```

Comment:
I'm afraid this is not tested and also not quite right.
While `torch.logical_or(torch.tensor(2), torch.tensor(1))` gives `tensor(True)`,
`torch.ops.aten.__or__(torch.tensor(2), torch.tensor(1))` gives `tensor(3)`
(and `torch.tensor(2) or torch.tensor(1))` gives `tensor(2)`, so it is different yet)

Reply:
Removed aten::__or__ from the convert_map.

Reply:
Thank you. Seems good to me now.